examples/3d/solari.rs

```diff
@@ -78,7 +78,7 @@ fn setup(
     commands
         .spawn((
             SceneRoot(asset_server.load(
-                GltfAssetLabel::Scene(0).from_asset("models/PicaPica/pica_pica_-_robot_01.glb"),
+                GltfAssetLabel::Scene(0).from_asset("https://github.com/bevyengine/bevy_asset_files/raw/refs/heads/main/picka_pica/robot_01.glb")
```

Comment:
```suggestion
                GltfAssetLabel::Scene(0).from_asset("https://github.com/bevyengine/bevy_asset_files/raw/2a5950295a8b6d9d051d59c0df69e87abcda58c3/pica_pica/robot_01.glb")
```
Fixed typos + use a fixed url